test/system/100-inspect.bats

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bats
+
+load helpers
+load helpers.registry
+load setup_suite
+
+# bats test_tags=distro-integration
+@test "ramalama inspect GGUF model" {
+    run_ramalama inspect tiny
+
+    is "${lines[0]}" "tinyllama" "model name"
+    is "${lines[1]}" "   Path: /home/mengel/.local/share/ramalama/models/ollama/tinyllama:latest" "model path"
```

Comment:
/home/mengel -> $HOME

Reply:
Used a wildcard to depend on the home directoy: `Path: .*models/ollama/tinyllama:latest`